packages/apps-config/src/endpoints/productionRelayKusama.ts

```diff
@@ -24,6 +24,7 @@ export function createKusama (t: TFunction): EndpointOption {
       OnFinality: 'wss://kusama.api.onfinality.io/public-ws',
       // 'Geometry Labs': 'wss://kusama.geometry.io/websockets', // https://github.com/polkadot-js/apps/pull/6746
       Dwellir: 'wss://kusama-rpc.dwellir.com',
+      RadiumBlock: 'wss://kusama.public.curie.radiumblock.co/ws',  
```

Comment:
```suggestion
      RadiumBlock: 'wss://kusama.public.curie.radiumblock.co/ws',
```

Remove trailing spaces (should make the CI lint happy)